openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -247,7 +247,7 @@ OpenAssessment.ResponseView.prototype = {
      array of strings: The current response texts.
      **/
     response: function(texts) {
-        var sel = $('.submission__answer__part__text__value', this.element);
+        var sel = $('form.response__submission .submission__answer__part__text__value', this.element);
```

Comment:
Instead of relying on the form HTML component (which could appear anywhere) to provide the scoping, I'd feel better if the scoping was done via a class that is specific to the response HTML. For instance, could openassessment__response be used to scope this?